handbook/engineering/search/goals.md

```diff
@@ -2,64 +2,52 @@
 
 ## Goals
 
-Goals help us achieve our [vision](index.md#vision) for search.
+### Support large scale organizations
 
-### Mature query language
+Timeline: ~6 months (May 2021)
 
-**Problem:** The Sourcegraph query language does not support the growing complexity of searches being run by our users.
+#### Definition of large scale
 
-Examples:
+Organizations with:
 
-- I want to only show examples from repositories that have been modified in the last 3 months so I see the most recent solutions.
-- I want to find repositories that have not been modified in over a year so I can deprecate them.
-- TODO: add more examples.
+- A Git directory size between 100 GB and 50 TB
+- Total lines of code (LOC) between 1 billion and 5 billion
```

Comment:
Should we explicitly call out differences between monorepos vs lots of repos? The sort of solutions that help monorepos don't normally help lots of repos. I would think focussing on monorepos (or possibly having a part of the team focus on monorepos) would make a lot of sense.

Although anything new we make needs to support both use cases (eg if we did indexed commit search).

Reply:
Good call out. We used Git dir size and LOC as  metrics (versus number of repos) because it felt like a more accurate measure of scale. We should capture the problems unique to `monorepos` vs `lots of repos`. Adding it to the `Problem` section as a specific group of problems we want to target solving seems to make sense. This would mean adding it as an item to the list Christina outlines here [comment](https://github.com/sourcegraph/about/pull/2001#discussion_r523891652).